### PR TITLE
Add minutes of Mentoring Subcommittee on 2015-04-10

### PR DESCRIPTION
**Pull request by r-gaia-cs**
@BillMills, @chendaniely, @karinlag and @mckays630 Could you give a +1 if you are OK with this?

### CODE REVIEW

File: subcommittees/mentoring/minutes-2015-04-10.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+    Screen record of how to setup workshops now lives at: http://swcarpentry.github.io/slideshows/creating-website/index.html#slide-0
+
+    It can be accessed directly from the Lessons page.
+
+-   KL: write draft proposal for "Study New Instructor Experience"
+
+    KL thinks she volunteered to write up a time plan for what mentors should do, am I mistaken?
+
+    Yes. --Raniere
+
```
Comment (karinlag) [inline subcommittees/mentoring/minutes-2015-04-10.md]: I forgot to edit this part, but I am writing up a mentoring guideline proposal.

File: subcommittees/mentoring/minutes-2015-04-10.md [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+    We, the Mentoring Subcommittee, have the final answer at this decision.
+
+    SM: The discussion of branding here kind of muddies the waters for me with respect to the specific charge of this subcommittee, and how that relates to the other subcommittees.  
+
+-   Options:
+
+    -   Mentoring
+
+        Suggested by Steering Committ
+
```
Comment (karinlag) [inline subcommittees/mentoring/minutes-2015-04-10.md]: Steering Committee

**Comment by r-gaia-cs** [comment]
> This is probably not useful now that the meeting is over.

We want to archive the minutes.

File: subcommittees/mentoring/minutes-2015-04-10.md [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+    Done. https://github.com/swcarpentry/board/pull/27
+
+-   TT: ask Greg about the screen record of how to setup workshop homepages.
+
+    Screen record of how to setup workshops now lives at: http://swcarpentry.github.io/slideshows/creating-website/index.html#slide-0
+
+    It can be accessed directly from the Lessons page.
+
+-   KL: write draft proposal for "Study New Instructor Experience"
+
```
Comment (karinlag) [inline subcommittees/mentoring/minutes-2015-04-10.md]: Remove this one.

File: subcommittees/mentoring/minutes-2015-04-10.md [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+    Done. https://github.com/swcarpentry/board/pull/27
+
+-   TT: ask Greg about the screen record of how to setup workshop homepages.
+
+    Screen record of how to setup workshops now lives at: http://swcarpentry.github.io/slideshows/creating-website/index.html#slide-0
+
+    It can be accessed directly from the Lessons page.
+
+-   KL: write draft proposal for "Study New Instructor Experience"
```
Comment (karinlag) [inline subcommittees/mentoring/minutes-2015-04-10.md]: Replace with:

KL will write proposal for mentorship guidelines/program

File: subcommittees/mentoring/minutes-2015-04-10.md [hunk 5]
```diff
@@ -0,0 +1,288 @@
...
+
+        Sheldon likes this (but wants 'instructor' in front of XXX subcommittee)  .
+
+        BM: Sounds like more Silicon Vally and business. (This is subjective -- I don't get a corporate read)
+
+    -   Community
+
+        Suggested by KL
+
+        KL: Should like bi-direction.
```
Comment (karinlag) [inline subcommittees/mentoring/minutes-2015-04-10.md]: Replace with:

KL: community suggests bi-directed communication

**Comment by karinlag** [comment]
+1 from me with the two changes I suggested.

**Pull request merged**